modules/appeals_api/app/swagger/higher_level_reviews/v0/swagger.json

```diff
@@ -55,7 +55,7 @@
           {
             "in": "header",
             "name": "X-VA-ICN",
-            "required": false,
+            "required": true,
             "description": "Veteran's ICN",
```

Comment:
These JSON files are auto-generated by rswag, so they shouldn't be edited directly. You can regenerate docs by using the rswag rake tasks - `bundle exec rswag:appeals_api:all` will regenerate everything at once, but it takes a while - there are api-specific tasks that are faster to run when updating the specs (`bundle exec rake -T` shows them all).

If you run those tasks right now, the specs will (correctly) fail with 422 errors, since they don't have the ICN header yet. I actually fixed a few instances of this same problem a while back for [Contestable Issues](https://github.com/department-of-veterans-affairs/vets-api/pull/12033/commits/ddfe8b7c563fe0dd6e0aa74b9660aa8203e07dac) and [Legacy Appeals](https://github.com/department-of-veterans-affairs/vets-api/pull/12033/commits/26a7cf306d4bb484d28fdf38e1e9a0b07e855dea) - the changes will be really similar for the HLR/NOD/SC docs specs (those commits also include some oauth-related changes, which aren't relevant to this). I'm happy to walk through this in slack and/or give an overview the rswag setup at our engineering sync some week, too.

Reply:
Thank you, @caseywilliams! I'm working on that now.